### DISCUSSION TITLE
When only the dismiss parameter is passed to useInteractions, does getReferenceProps still perform a meaningful operation?

### DISCUSSION THREAD

**Discussion by SEOKKAMONI** [floating-ui/floating-ui#2725] [Q&A]
```tsx
 const dismiss = useDismiss(context);
 const role = useRole(context, { role: 'tooltip' })

  const {getReferenceProps, getFloatingProps} = useInteractions([
    dismiss,
    role
  ]);
```

When the code is as follows, is it unnecessary to have getReferenceProps? Please explain.

**Comment by atomiks** [comment]
Well if you have `useRole` as well, then yes. Otherwise, `getReferenceProps` is necessary for the `bubbles` option and `referencePress` option in certain scenarios. It's probably a good idea to always spread it to prevent issues in the future.

**Reply by SEOKKAMONI** [reply]
Thank you for your kind explanation
